package.json

```diff
@@ -25,7 +26,6 @@
   "dependencies": {
     "jstransformer-markdown-it": "^2.1.0",
     "marked": "^4.0.12",
-    "purs": "^0.3.0",
```

Comment:
Was this intentionally a dependency? I don't think that `purs` is maintained, and this seems to have a dependency on a very old version of `purescript` (I was seeing `purescript@12.5` installing, I think) https://github.com/shinnn/purs

Reply:
looks like it's not a real dep, thanks for spotting it :)